app/src/main/java/com/example/dndwiki/repository/Preferences.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.dndwiki.repository
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class Preferences(context: Context) {
+
+    private val preferences: SharedPreferences = context.getSharedPreferences("sharedPref", Context.MODE_PRIVATE)
+    private val editor: SharedPreferences.Editor = preferences.edit()
+
+    fun saveData(saveDate: Long) {
+
+        editor.apply {
+            putLong("LONG_KEY", saveDate)
```

Comment:
LONG_KEY is probably not the best of keys to use if you need to keep this unique in the future and possibly will add more things :)